Makefile

```diff
@@ -1,23 +1,36 @@
 # SONiC make file
 
 NOJESSIE ?= 0
+NOSTRETCH ?= 0
```

Comment:
>NOSTRETCH [](start = 0, length = 9)

The NOSOMETHING macros are counter-intuitive. Seems more and more coming.

Reply:
this is not related to buster upgrade. will triage this in the normal dev process, not bundle to buster upgrade.